apps/nextevent/manifest.yaml

```diff
@@ -0,0 +1,8 @@
+---
+id: next-event
+name: Next Event Date
+summary: Displays next event only
+desc: Displays the next calendar event, no matter how far away, from iCal (.ics) files.
```

Comment:
```suggestion
desc: Displays the next calendar event, no matter how far away, from iCal (.ics) files. Note: This app will process your iCal data via an external service outside of the Tidbyt ecosystem.
```

Reply:
A colon was breaking yaml syntax and causing the build to fail. I added another commit after this to remove it from the note and replace with a semicolon.